### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ramiaquista**

### CODE REVIEW

**Comment by ramiaquista** [comment]
STATUS: APPROVED 🎉✔

**Comment by ramiaquista** [comment]
### **STATUS: APPROVED 🎉✔**

Your project is complete and awesome! There is nothing else to say. It's time to merge it!

**Optional Changes:**

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

       - Adding a live demo link in your Readme file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+  <main id="main">
+    <h1 id="title">Diego Armando Maradona</h1>
```
Comment (ramiaquista) [inline index.html:11]: - [ ] OPTIONAL: You could wrap this h1 tag inside in a header tag to make your code clean! 🧹

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <strong>2000</strong> Collapses with heart problems
+          </li>
+          <li>
+            <strong>2020</strong> Maradona dies in his house.
+          </li>
+      </ul>
+      <blockquote cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
+          <p>
+            "Maradona was a great football player, he won the 1986 World Cup with Argentina Football Team. He won 2 times the Seria-A ligue with Napoli football club."
+          </p>
```
Comment (ramiaquista) [inline index.html:55]: - [ ] OPTIONAL: You could separate in differents lines your p text in order to be more clean and easy to read what is inside!

**Review by ramiaquista — COMMENTED** [review]
STATUS: APPROVED 🎉✔
Your project is complete and awesome! There is nothing else to say. It's time to merge it!

Optional Changes:

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

-Adding a live demo link in your Readme file

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**